### ISSUE TITLE
Invalid return type from ImageStorage::getNoImage 

### ISSUE THREAD

**Issue by jfilla** [ublaboo/image-storage#19]
error in ImageStorage:284
```
TypeError : Return value of Contributte\ImageStorage\ImageStorage::getNoImage() must be an instance of Contributte\ImageStorage\Image, array returned
```
version: v2.0.0 
hash: 70b37fbda47d00e3346f4c3b2425c63cbd77bf27

**Comment by rozsival** [comment]
@paveljanda This seems to be resolved with latest `2.0.3` release.

**Comment by paveljanda** [comment]
It should be indeed 👍
